### ISSUE TITLE
Train LeNet CNN on the training data from the BabyAIShapes using LRP toolbox

### ISSUE THREAD

**Issue by elboyran** [NLeSC/XAI#4]
Use the validation and testing sets for evaluating performance
Use [cnn_training_testing](https://github.com/NLeSC/XAI/blob/master/Software/lrp_toolbox-1.2.0/matlab/cnn_training_test.m) as a starting point 

- [x] write a function which normalizes the input data a required by LeNet networks

- [x] function to reshape labels as required by LRP toolbox - from labels to vectors of occurance

- [x] CNN: LeNet architecture module

- [x] train the CNNusing training and validation data and test on the test data using the LRP Toolbox

- [x] save the trained model with good performance

**Comment by elboyran** [comment]
The model  has been trained on the train set with 50 000 iterations achieving accuracy of 99.64% on the validation set.
